src/features/users/store/store.ts

```diff
@@ -0,0 +1,31 @@
+import { ZetkinUser } from 'utils/types/zetkin';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RemoteItem, remoteItem } from 'utils/storeUtils';
+
+export interface UsersStoreSlice {
+  userItem: RemoteItem<ZetkinUser>;
+}
+
+const initialUsersState: UsersStoreSlice = {
```

Comment:
I think this should just be called `initialState`, like in other `store` modules.